### PR TITLE
[Python] Challenge 6 (Unreviewed)

### PR DESCRIPTION
**Pull request by sarcodian**

### CODE REVIEW

File: challenge_6/python/sarcodian/src/challenge_6.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+                temp_list.append(int_list[i])
+                final_list.append(print_range(temp_list))
+        elif int_list[i] == int_list[i+1]-1:
+            temp_list.append(int_list[i])
+        else:
+            temp_list.append(int_list[i])
+            final_list.append(print_range(temp_list))
+            temp_list.clear()
+    print(final_list)
+    return final_list
```
Comment (ningyuansg) [inline challenge_6/python/sarcodian/src/challenge_6.py]: Style-wise, having both a `print` and `return` seems redundant to me.

File: challenge_6/python/sarcodian/src/challenge_6_tests.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+class TestBinaryMethods(unittest.TestCase):
+
+    # All tests must begin with the test_* naming scheme otherwise unittest won't find it
+    def test_positiveIntegers(self):
+        """ Tests from the READ.me """
+        # Testing input with only 1 return value
+        self.assertEqual(ranges([1, 2]), ["1->2"])
+
+# Testing input with no return value
+        self.assertEqual(ranges([1]), [])
```
Comment (ningyuansg) [inline challenge_6/python/sarcodian/src/challenge_6_tests.py]: Style-wise, it is better to contain each test within their own method. You should name the method something representative of the type of test you are running, such as `test_single_element`.

File: challenge_6/python/sarcodian/src/challenge_6.py (hunk 1 above)
Comment (sarcodian) [inline challenge_6/python/sarcodian/src/challenge_6.py]: prolly a debug print. will remove.

File: challenge_6/python/sarcodian/src/challenge_6_tests.py (hunk 2 above)
Comment (sarcodian) [inline challenge_6/python/sarcodian/src/challenge_6_tests.py]: for clarification, method in testing is the def test_somthing(): part?

**Comment by sarcodian** [comment]
Have added documentation.

File: challenge_6/python/sarcodian/src/challenge_6_tests.py (hunk 2 above)
Comment (ningyuansg) [inline challenge_6/python/sarcodian/src/challenge_6_tests.py]: Yep

**Comment by ningyuansg** [comment]
@sarcodian I think you forgot to `git add` your documentation in the commit 9f31a75.

**Comment by sarcodian** [comment]
@ningyuansg Added the docstrings. Couldn't think of anything to say in the READ.me

**Pull request merged**